kit/return.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: Kit Return
+---
+
+# Kit Return
+
+All items [provided](/docs/kit/) in the kit, with the exception of wire, must be returned at the end of the competition. The Kit List document included with your kit has a list of items included in the kit.
+
+After the competition event, you will need to extract the kit from your robot and return them. If you are not using every component of the kit, it should still be brought to the competition so it can be returned. Your returned kit will be checked as you leave the venue. If you have forgotten any items, they must be returned as soon as possible after the event.
+
+<div class="info">
+Be sure to take all of your own kit with you - it may not be possible to return forgotten items to you promptly.
+</div>
+
+## Kit Shipping
+
+If you are unable to return your kit at the competition, it must be shipped to us at <a href="{{ site.url }}/contact/">this address</a>, unless otherwise instructed.
```

Comment:
```suggestion
If you are unable to return your kit at the competition, it must be shipped to us at [this address]({{ site.url }}/contact/), unless otherwise instructed.
```

Reply:
However I'm not sure if that url is right? If this is meant to point to https://studentrobotics.org/ we prefer to make that explicit rather than using `{{ site.url }}`.

Reply:
I copied it from the footer links, which use the same code.